index.html

```diff
@@ -3,87 +3,82 @@
 	
 	<head>
 		<title>Charlotte Buckley - Portfolio</title>
-		<link rel="stylesheet" type ="text/css" href="style.css"> 
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
 		<link rel="stylesheet" type="text/css" href="normalize.css">
+		<link rel="stylesheet" type ="text/css" href="style.css"> 
 		<link rel="preconnect" href="https://fonts.googleapis.com">
 		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
 		<link href="https://fonts.googleapis.com/css2?family=Varela+Round&display=swap" rel="stylesheet">
 	</head>	
 	
 	<body>		
-		<nav class="nav-bar">
+		<nav class="nav-bar" id="top">
 			<a class="navigation-links" href="#home">home</a> 
 			<a class="navigation-links" href="#about-me">about me</a>
-			<button class="cv-button">download my CV!</button>
+			<img src="c.png" alt="CB Logo">
```

Comment:
alt text not really describing the image, how about 'letters cb entwined with beautiful flowers'